packages/e2e-tests/tests/breakpoints-04.test.ts

```diff
@@ -35,6 +36,7 @@ test(`breakpoints-04: catch, finally, generators, and async/await`, async ({
 
   async function resumeToBreakpoint(page: Page, lineNumber: number) {
     await addBreakpoint(page, { lineNumber, url: exampleKey });
+    await delay(200);
```

Comment:
No objection to this but curious what prompted this change? Seems unrelated to ThreadPoint :)

Reply:
This is indeed unrelated and I'm still looking into it - the last successful run of breakpoints-04 was more than a week ago and one reason we didn't notice it is that most recordings of that test fail to upload, so most test runs don't contain a single recording of it.

Reply:
SCS-1540